app/helpers/link_to_helper.rb

```diff
@@ -273,4 +287,12 @@ def message_path(message)
     message_url(message, :only_path => true)
   end
 
+  def format_event_label(label, is_script)
```

Comment:
It would be worth adding a line of documentation about why you might need to use this.